text-ui-test/EXPECTED.TXT

```diff
@@ -1,4 +1,10 @@
-Hello
-Spaghetti Carbonara / LUNCH
+-----------------------------
+Welcome to Recipe.io!
 What is your name?
-Hello Yuki
+Yuki
```

Comment:
I think the expected file shouldn't have this. "Yuki" is an input, not an output